### ISSUE TITLE
Multiple On-Use Trinket (And Belt) Cooldowns

### ISSUE THREAD

**Issue by navv1234** [simulationcraft/simc#2580]
Originally reported on Google Code with ID 2581
```
Mirror of the Blademaster does not share a cooldown with other on-use trinkets in-game,
which simcraft does not currently model.

Another related issue is that belt tinkers are putting on-use trinkets on cooldown,
when there should be no interaction between them. The linked profile does not use any
trinket on the pull, because they are on a shared CD from using the Grounded Plasma
Tinker.


Example Profile: http://pastebin.com/Hc4QryBW
```

Reported by `Kungersma` on 2015-07-08 08:24:02

**Comment by navv1234** [comment]
This is now partially supported (item shared cooldown system). I need to look at the mechanisms behind addons to figure out how they relate to this, as they are not statically assigned to item data.

**Comment by navv1234** [comment]
This is now (in my opinion) fully fixed in 36e0c6d. Engineering on-use effects (or more generally, on-use effects from a non-item source) are no longer included in default APLs, and as such, they cannot prevent potion use. They can still be manually activated as one would expect (`use_item,slot=waist`).
